niimprint/models.py

```diff
@@ -0,0 +1,33 @@
+from dataclasses import dataclass
+
+
+@dataclass
+class Device:
+    name: str
+    max_density: int
+    max_width: int
+
+
+SUPPORTED_DEVICES = {
+    "D11": Device("D11", 3, 96),
+    "D110": Device("D110", 3, 96),
+    "B1": Device("B1", 5, 384),
+    "B21": Device("B21", 5, 384),
+}
+
+
+class RejectionError(BaseException):
```

Comment:
Assert / RuntimeError is probably fine for now... or add a generic `NiimPrintError`